docs/index.md

```diff
@@ -15,8 +15,7 @@ hide:
 
 <div class="bitovi-row">
     <div class="bitovi-column">
-        <h1>Organize your Infrastructure-as-Code (IAC) 
-        <nl>Consistently Deploy.</h1>
+        <h1>BitOps packages, centralizes and organizes IaC</h1>
```

Comment:
Would that work?
```suggestion
        <h1>BitOps packages, centralizes and organizes your Infrastructure-as-Code</h1>
```
I think it's nice to spell the IaC explicitly as we're targeting the entry group where the `IaC` abbreviation could be not that clear.